### PR TITLE
docs(project): add project specific docs

### PR DESCRIPTION
**Pull request by utkarshmani1997**
Signed-off-by: Utkarsh Mani Tripathi <utkarsh.tripathi@mayadata.io>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+[A clear and concise description of what the bug is, and what commands you ran.)
+
+
+**What did you expect to happen:**
+
+
+**The output of the following commands will help us better understand what's going on**:
+(Pasting long output into a [GitHub gist](https://gist.github.com) or other pastebin is fine.)
+
+* `kubectl logs <jiva-operator pod name> -n openebs`
```
Comment (kmova) [inline .github/ISSUE_TEMPLATE/bug_report.md:17]: should this include kubectl get of the jivavolume CR, kubectl get SC and any other objects that should be fetched to debug.

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+## Release Candidate Verification Checklist
+
+Every release has release candidate builds that are created starting from the third week into the release. These release candidate builds help to freeze the scope and maintain the quality of the release. The release candidate builds will go through:
+- Platform Verification
+- Regression and Feature Verification Automated tests.
+- Exploratory testing by QA engineers
+- Strict security scanners on the container images
+- Upgrade from previous releases
+- Beta testing by users on issues that they are interested in.
+- Dogfooding on workload and MayaData OpenEBS Director staging cluster.
```
Comment (kmova) [inline RELEASE.md:13]: This has been changed to mention about OpenEBS E2e cluster and not use director.

Comment (utkarshmani1997) [reply RELEASE.md:13]: done

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (utkarshmani1997) [reply .github/ISSUE_TEMPLATE/bug_report.md:17]: done

**Pull request merged**